discovery-infra/start_discovery.py

```diff
@@ -345,16 +345,19 @@ def _cluster_create_params():
         "vip_dhcp_allocation": bool(args.vip_dhcp_allocation) and not user_managed_networking,
         "additional_ntp_source": ntp_source,
         "user_managed_networking": user_managed_networking,
-        "high_availability_mode": "None" if args.master_count == 1 else "Full"
+        "high_availability_mode": "None" if args.master_count == 1 else "Full",
```

Comment:
Use https://github.com/openshift/assisted-test-infra/blob/ea04ad57551b67bdf1e057bfe808b8c11514fefd/discovery-infra/test_infra/consts.py#L151-L153

Reply:
I only added `,` at the end. I do not want to change it

Reply:
> I only added `,` at the end. I do not want to change it

Right, missed that.